app/docker/docker-compose.selenium.yml

```diff
@@ -10,8 +10,8 @@ services:
       com.centurylinklabs.watchtower.enable: 'true'
     shm_size: '2gb'
     environment:
-      - "START_XVFB=true"
-      - "TZ=${DAB_APPS_SELENIUM_TZ:-UTC}"
+      START_XVFB: 'true'
+      TZ: '${DAB_APPS_SELENIUM_TZ:-UTC}'
```

Comment:
If you are using an env car in this string then you need to use double quotes instead otherwise it is treated literally.